operations/finance.md

```diff
@@ -1486,6 +1492,7 @@ Adds a new external payment to a bill of the specified customer. An external pay
 | `BillId` | string | optional | Unique identifier of an open bill of the customer where to assign the payment. |
 | `Amount` | [Amount value](finance.md#amount-value) | required | Amount of the external card payment. |
 | `Type` | string [External payment type](finance.md#external-payment-type) | optional | Type of the external payment. |
+| `ExternalIdentifier` | string | optional | Identifier of the external payment accepted as a pre-payment. |
```

Comment:
This is way too specific description, this can be used in other way than just the one case from the issue.
```suggestion
| `ExternalIdentifier` | string | optional | Identifier of the payment from external system. |
```